### ISSUE TITLE
bug-bogue:  In Gmail, Voiceover does not read the French properly

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#1745]
##When using French language tags to send email messages to recipients with Gmail addresses, once the recipient receives the email, if they're using Voiceover then Gmail reads the French with English pronunciation.

## [SEV-3 Minor] I seem to not have access to the Bug Severity doc and have requested access. For the time being,  listing this as minor because our A11y statement explicitly states in a heading:
SEV-3 Minor

## To Reproduce

1. Create a template with language tags in GC Notify or use this template: e8f385c9-7a89-4c88-bc4d-7d72464cf5a1
Could also use this text if you prefer to create a new template:
[[en]]
Today is Wednesday
[[/en]]
[[fr]]
Aujourd'hui c'est mercredi
[[/fr]]
2. Save the template and then send the message to a Gmail address that you can access.
3. Enable Voiceover on your computer and listen to the message
4. Voiceover reads the French with English pronunciation.

## Expected behavior

When someone using Voiceover receives an email message with French content in their Gmail, Voiceover will read it with English pronunciation.

## Impact

Recipients who use Voiceover in Gmail may not be able to understand French content. 

If applicable

Impact on Notify users: Cannot effectively communicate messages to Francophone or French-first recipients who depend on Gmail and Voiceover.

Impact on Recipients: This will have a detrimental impact on recipients using Voiceover who need to understand the message in French (Francophone or French-first recipients)

Impact on Notify team: We have already documented in our A11y statement that we cannot control the accessibility of end-user messages.
